doc/rbd/rados-rbd-cmds.rst

```diff
@@ -4,7 +4,7 @@
 
 .. index:: Ceph Block Device; image management
 
-The ``rbd`` command enables you to create, list, introspect and remove block
+The ``rbd`` command enables you to create, list, inspect and remove block
```

Comment:
Not sure I follow what is wrong with "introspect", but fine with me.

Reply:
>examine one's own thoughts or feelings.

I agree with this change.  "Introspect" is a bit elliptical for ESOL speakers.